holoviews/element/util.py

```diff
@@ -52,6 +43,54 @@ def compute_edges(edges):
     return np.concatenate([edges, [edges[-1]+width]])
 
 
+def compute_slice_bounds(slices, scs, shape):
```

Comment:
I'm not sure I would recommend adding tests for this utility right now as I do want to merge ASAP. At least it is much easier to test now!